mlrun/feature_store/retrieval/base.py

```diff
@@ -653,7 +679,7 @@ def _get_engine_df(
         column_names: typing.List[str] = None,
         start_time: typing.Union[str, datetime] = None,
         end_time: typing.Union[str, datetime] = None,
-        entity_timestamp_column: str = None,
+        time_column: str = None,
```

Comment:
```suggestion
        time_column: Optional[str] = None,
```

Reply:
@gtopper As I understand it 'time_column: Optional[str]' and 'time_column: str = None' are equal no?